zeeguu/core/definition_of_learned/is_learned.py

```diff
@@ -3,6 +3,9 @@
 
 def is_learned_based_on_exercise_outcomes(exercise_log):
     """
+    10-06-2024 -> OUTDATED, now words are considered learned after the user
+    has completed a learning cycle and depeding on how the productive flag is set.
+
```

Comment:
should we remove this then? it seems it's only called from one other function `update_learned_status` which is only called from one test. We can remove this, the function and the test, and nothing's lost. Or?

Reply:
I do think it's safe to remove, but I don't feel comfortable removing methods from the codebase without your approval!